### PR TITLE
[TIMOB-18414] Update min Node version to 0.10

### PR DESCRIPTION
**Pull request by skypanther**
https://jira.appcelerator.org/browse/TIMOB-18414

### CODE REVIEW

**Comment by jhaynie** [comment]
we should add 0.12 now that it's stable.

File: README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ powerful and extensible enough for production usage.
 
 ## Prerequisites
 
-The Titanium CLI requires [Node.js 0.8.x](http://nodejs.org/dist/). You can
-download Node.js 0.10.x, but only Titanium SDK 3.1.0 and later will work.
+The Titanium CLI requires [Node.js 0.10.x](http://nodejs.org/dist/) or newer. Versions
+of Titanium earlier than Titanium SDK 3.1.0 require Node 0.8.x.
```
Comment (cb1kenobi) [inline README.md]: I think you can just omit all mention of Node.js 0.8.

Comment (jhaynie) [inline README.md]: :clap:

**Comment by skypanther** [comment]
@jhaynie In testing this morning, I found that the CLI isn't compatible with 0.12. I've opened TIMOB-18538, which I'll be looking into resolving asap.

**Comment by jhaynie** [comment]
even better. switch it on and force the build to be in flames until we fix it. :fire:

**Comment by feons** [comment]
**Approved.**

**Pull request merged**